app/grandchallenge/publications/models.py

```diff
@@ -147,3 +147,11 @@ def ama_html(self):
         citation = re.sub(r"^1\. ", "", citation)
 
         return clean(citation)
+
+    @property
+    def authors(self):
+        authors = []
+        match = re.search(r".+?\.", self.citation)
+        for author in re.split(",", match.group()):
+            authors.append(re.split(r"\s", re.sub(r"^\s", "", author))[0])
```

Comment:
Rather than parsing the citation which could change it will be quicker and more robust to extract this from the `csl`. Something like `authors = [a["family"] for a in self.csl.get("author", [])]`

Reply:
Yes, this was how I initially tried it, but I believe that the csl does not only contain the authors of the paper but also authors of papers cited in the paper, and they are all under `authors`, as far as I could tell, so that's why I resorted to extracting the authors from the citation. I will take another look though.

Reply:
Ah you were completely right, your query above works like a charm. Thank you!